SPECS/kata-containers/kata-containers.spec

```diff
@@ -97,6 +97,7 @@ tar -xf %{SOURCE1}
 %build
 export PATH=$PATH:"$(pwd)/go/bin"
 export GOPATH="$(pwd)/go"
+export OPENSSL_NO_VENDOR=1
```

Comment:
in this SPEC we export the variables in the %install section as well.

in the kata-containers-cc SPEC we do not export the variables.

Reply:
To clean up as part of next iteration